src/renderer/css/global.css

```diff
@@ -1,24 +1,24 @@
 body {
-    user-select: none;
+	user-select: none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5444487)